spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -267,7 +267,47 @@ Collections of embedded objects can also be stored as the values of embedded Map
 
 ==== Relationships
 
-This feature is not supported yet.
+Parent-child relationships are supported via the `@Descendants` annotation.
+
+[source, java]
+----
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Descendants;
+import org.springframework.data.annotation.Id;
+
+@Entity
+public class ShoppingOrder {
+  @Id
+  long id;
+
+  @Descendants
+  List<Item> children;
+}
+
+@Entity
+public class Item {
+  @Id
+  Key purchasedItemKey;
+
+  String name;
+
+  Timestamp originalAcquisitionTime;
```

Comment:
I was expecting something like price or count. What's that?
```suggestion
  BigDecimal price;
```

Reply:
@meltsufin  We cannot put "price" here because it is contrary to the example. 

I specifically avoided doing that. 

Price is something "static" for all items of that price, and as you can see our item is specific to a specific shopping cart. "originalAcquisitionTime" is the time this specific physical item was acquired from the factory or supplier, and that is not static to every item of the same model or type.

Reply:
Maybe I don't understand retail enough, but I never considered that an item in the shopping cart has some acquisition time attirbute. Maybe time of placement into the cart? Maybe a realistic example would be to have a reference to a product? I don't know, I'm confused by this example. Does it make sense to everyone else?

Reply:
This really doesn't have anything to do with retail. It just needs to be an attribute specific to an instance of an item, and not just to the model of item. 

If it helps you understand, I can change this the time it was added to cart instead of time the item entered the warehouse.